src/pages/library/DataBrowser.js

```diff
@@ -113,6 +113,8 @@ const SelectedItemsDisplay = ({ selectedData, setSelectedData }) => {
         style: { textTransform: 'none', fontSize: 14 },
         onClick: () => {
           Ajax().Metrics.captureEvent(`${Events.catalogWorkspaceLink}:tableView`, {
+            // These are still using snapshot as a relic to ensure backwards search
```

Comment:
I didn't want to change the name of the fields for these, because it would ruin searching and metrics in mixpanel, but added a comment just so it is clear.